### ISSUE TITLE
bug: PAGE_URL never changed

### ISSUE THREAD

**Issue by mortalYoung** [VincentLoy/share-selected-text#21]
Hey, I wonder are you still maintaining this project? 

'Cause I have a problem using this project.

After importing `share-selected-text/dist/shareSelectedText.js` in my private SPA project, I find that `PAGE_URL` will never changed.

Why not just get current url while using it rather than defining it as a const value?

For example 

```diff
- let twitterUrl = `https://twitter.com/intent/tweet?url=${PAGE_URL}&text="${text}"`;
+ let twitterUrl = `https://twitter.com/intent/tweet?url=${getPageUrl()}&text="${text}"`;
```

**Comment by VincentLoy** [comment]
Hi @mortalYoung, 

I think you are right, the plugin may have some problems for getting the page URL in SPA projects. Your suggestion sounds great.
I'll take a look.

Thanks for your suggestion !

**Comment by VincentLoy** [comment]
@mortalYoung , I just fixed it in version v1.1.5 ! Thanks !

**Comment by mortalYoung** [comment]
> @mortalYoung , I just fixed it in version v1.1.5 ! Thanks !

Thanks a lot ! :)
